scripts/bb_sim.py

```diff
@@ -314,4 +346,9 @@ def unfinished():
     bin_data.seek(bin_seek_vsite[i])
     vs30.tofile(bin_data)
 bin_data.close()
+logger.debug("Completed {} stations".format(len(stations_todo)))
+
 print("Process %03d of %03d finished (%.2fs)." % (rank, size, MPI.Wtime() - t0))
```

Comment:
not logger?

Reply:
The last "print" is intentional as it will add to .out and give an easy visibility when the slurm job has completed.